pkg/volume/iscsi/iscsi.go

```diff
@@ -198,6 +199,7 @@ type iscsiDisk struct {
 	chap_discovery bool
 	chap_session   bool
 	secret         map[string]string
+	InitiatorName  string
```

Comment:
@liggitt 
The persistISCSI() save iscsiDisk struct to json file and the loadISCSI() get iscsiDisk struct from json file.

- persistISCSI()
https://github.com/mtanino/kubernetes/blob/e4ff66081037bb970e087722d6cba609ee4b2efb/pkg/volume/iscsi/iscsi_util.go#L164

- loadISCSI()
https://github.com/mtanino/kubernetes/blob/e4ff66081037bb970e087722d6cba609ee4b2efb/pkg/volume/iscsi/iscsi_util.go#L178

Is this answer for the question?